Src/BootCamp.Chapter/Program.cs

```diff
@@ -4,9 +4,34 @@ namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
+            var logger = new Logger();
+            logger.Log("Program boots.");
 
+            for (int num = 0; num < 2; num++)
+            {
+                Console.WriteLine("Calculate BMI for a person.");
```

Comment:
Please apply what you learning in Chapter 1 / homework 3- use functions. The body of this loop should be a function to PromptBmi

Reply:
As requested, I have created a PromptBmi function when prompting for BMI input.